elide-integration-tests/src/test/java/com/yahoo/elide/tests/ResourceIT.java

```diff
@@ -2670,6 +2670,7 @@ public void testSpecialCharacterLikeQueryHQL(FilterPredicate filterPredicate, in
         when(pagination.isGenerateTotals()).thenReturn(true);
         tx.loadObjects(EntityProjection.builder()
                 .type(Book.class)
+
```

Comment:
There are some other graphQL things I commented out in the base branch.  For example, I removed the graphQL endpoint from JAX-RS.  We'll need to undo all of those so we don't have a regression.